### ISSUE TITLE
PHP requirements don't match Magento 2.4.4

### ISSUE THREAD

**Issue by Axel29** [Strategery-Inc/Magento2-InfiniteScroll#70]
### Preconditions

1. Magento 2.4.4

### Steps to reproduce

1. Run `composer require strategery/magento2-infinitescroll`

### Expected result

1. No error is thrown
2. The module is installed

### Actual result

1. The following error is thrown because of PHP <= 7.4 in the composer.json requirements while Magento 2.4.4 requires PHP >= 8.1:

```bash
Package strategery/magento2-infinitescroll has requirements incompatible with your PHP version, PHP extensions and Composer version:                           
    - strategery/magento2-infinitescroll 0.5.2 requires php ~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0 which does not match your installed version 8.1.1.
```

**Comment by ihor-sviziev** [comment]
Hi @Axel29,
Thank you for the report!
Could you try installing the extension with ` --ignore-platform-reqs` flag and confirm it works?

**Comment by SerhiiMandrykin** [comment]
Hi. We also need support of php 8.1 and magento 2.4.5

**Comment by asanka-atapattu** [comment]
Hi,

I am facing the same issue 

> Package strategery/magento2-infinitescroll has requirements incompatible with your PHP version, PHP extensions and Composer version:                           
    - strategery/magento2-infinitescroll 0.5.2 requires php ~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0 which does not match your installed version 8.1.1.

Could you please update your module?
Thanks

**Comment by aleksinoleg** [comment]
Hi
I tried to install the extension with --ignore-platform-reqs flag.
It works fine for me.

**Comment by ihor-sviziev** [comment]
The issue is fixed in version 0.6.0
